test/helpers/contents_list_helper_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class ContentsListHelperTest < ActionView::TestCase
+  test "it wraps the number and text in span elements" do
```

Comment:
It will only match the pattern of a number followed by a period at the start of the text of the link. I've now added a test for this specific example though.